ontap/client.go

```diff
@@ -180,6 +180,7 @@ type Client struct {
 	VolumeUnmount          *VolumeUnmount
 	VserverGetIter         *VserverGetIter
 	VserverShowAggrGetIter *VserverShowAggrGetIter
+	QtreeListIter          *QtreeListIter
```

Comment:
it's sorted by the ABC, so better move it to according to it instead of adding it to the end of the list